yarn-project/merkle-tree/src/test/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { default as memdown, type MemDown } from 'memdown';
```

Comment:
Generally speaking, we shouldn't create files names things like 'utils'. You could simply call this append_leaves.ts

Reply:
Addressed in 9aad0af81a3b67f171f2a65db46651c12ee41fa9